packages/thirdweb/src/react/core/hooks/transaction/usePrepareTransaction.ts

```diff
@@ -48,10 +48,10 @@ export function usePrepareTransaction<
   const abi extends Abi,
   const params extends object,
 >(
-  extension: (
+  extension?: (
```

Comment:
Ugh, the regretssss. Don't see another way around it without breaking API though. Live and learn